pennylane/ops/qutrit/channel.py

```diff
@@ -222,3 +222,93 @@ def compute_kraus_matrices(p):  # pylint:disable=arguments-differ
+    r"""
+    Single-qutrit amplitude damping error channel.
+    Interaction with the environment can lead to changes in the state populations of a qutrit.
+    This can be modelled for qutrits by the qutrit amplitude damping channel, with the following Kraus matrices:
+
+    .. math::
+        K_0 = \begin{bmatrix}
+                1 & 0 & 0\\
+                0 & \sqrt{1-\gamma_1} & 0 \\
+                0 & 0 & \sqrt{1-\gamma_2}
+                \end{bmatrix}, \quad
+        K_1 = \begin{bmatrix}
+                0 & \sqrt{\gamma_1} & 0 \\
+                0 & 0 & 0 \\
+                0 & 0 & 0
+                \end{bmatrix}, \quad
+        K_2 = \begin{bmatrix}
+                0 & 0 & \sqrt{\gamma_2} \\
+                0 & 0 & 0 \\
+                0 & 0 & 0
+                \end{bmatrix}
+
+    Where :math:`\gamma_1 \in [0, 1]` and :math:`\gamma_2 \in [0, 1]` are the amplitude damping
+    probabilities for subspaces (0,1) and (0,2) respectively.
+
+    .. note::
+
+        The Kraus operators :math:`\{K_0, K_1, K_2\}` are adapted from [`1 <https://doi.org/10.48550/arXiv.1905.10481>`_] (Eq. 8).
+
+    **Details:**
+
+    * Number of wires: 1
+    * Number of parameters: 2
+
+    Args:
+        gamma_1 (float): :math:`|1 \rangle \rightarrow |0 \rangle` amplitude damping probability.
+        gamma_2 (float): :math:`|2 \rangle \rightarrow |0 \rangle` amplitude damping probability.
+        wires (Sequence[int] or int): the wire the channel acts on
+        id (str or None): String representing the operation (optional)
+    """
+
+    num_params = 2
+    num_wires = 1
+    grad_method = "F"
+
+    def __init__(self, gamma_1, gamma_2, wires, id=None):
+        super().__init__(gamma_1, gamma_2, wires=wires, id=id)
+
+    @staticmethod
+    def compute_kraus_matrices(gamma_1, gamma_2):  # pylint:disable=arguments-differ
+        r"""Kraus matrices representing the QutritAmplitudeDamping channel.
+
+        Args:
+            gamma_1 (float): :math:`|1\rangle \rightarrow |0\rangle` amplitude damping probability.
+            gamma_2 (float): :math:`|2\rangle \rightarrow |0\rangle` amplitude damping probability.
+
+        Returns:
+            list(array): list of Kraus matrices
+
+        **Example**
+
+        >>> qml.QutritAmplitudeDamping.compute_kraus_matrices(0.5, 0.25)
+        [
+        array([ [1.        , 0.        , 0.        ],
+                [0.        , 0.70710678, 0.        ],
+                [0.        , 0.        , 0.8660254 ]]),
+        array([ [0.        , 0.70710678, 0.        ],
+                [0.        , 0.        , 0.        ],
+                [0.        , 0.        , 0.        ]]),
+        array([ [0.        , 0.        , 0.5       ],
+                [0.        , 0.        , 0.        ],
+                [0.        , 0.        , 0.        ]])
+        ]
+        """
+        for gamma in (gamma_1, gamma_2):
+            if not (math.is_abstract(gamma_1) or math.is_abstract(gamma_2)):
```

Comment:
gamma_1 and gamma_2 are checked separately if they are abstract, I believe this would be the correct behaviour, but if  I am mistaken and both should be abstract if one is then please let me know and I will fix it.

Reply:
Good question - presumably both would be passed in the same interface, but I think it doesn't hurt to check both. However the for loop around the if statement has redundancy, either check one at a time or use the iterating variable from the loop. May also be good to include the explicit ID of any offending gammas so the user knows if it's 1 or 2.